formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -319,3 +319,28 @@ let myFun
    */
 /* Bucklescript FFI item attributes */
 external imul : int => int => int = "Math.imul" [@@bs.val];
+
+class type _x = {
+                  method height: int
```

Comment:
by default, lists are `break:Never`. It could be that some list needs to specify `~break:IfNeed`. Other than that, the lists that append to labels *typically* need to be `~inline:(true, false)` to wrap in pleasing ways. So look around `class_types` in the printer to see if any lists and/or labels sound like they could be the culprit.